ext/ddtrace.c

```diff
@@ -453,6 +453,7 @@ static void ddtrace_activate(void) {
     ddtrace_telemetry_rinit();
     zend_hash_init(&DDTRACE_G(traced_spans), 8, unused, NULL, 0);
     zend_hash_init(&DDTRACE_G(tracestate_unknown_dd_keys), 8, unused, NULL, 0);
+    zend_hash_init(&DDTRACE_G(git_metadata), 8, unused, NULL, 1);
```

Comment:
I think this now requires a destructor to free the memory (release the zend string and the struct itself).